spring-boot-project/spring-boot/src/main/java/org/springframework/boot/jdbc/DatabaseDriver.java

```diff
@@ -83,6 +83,19 @@ public enum DatabaseDriver {
 	 */
 	POSTGRESQL("PostgreSQL", "org.postgresql.Driver", "org.postgresql.xa.PGXADataSource", "SELECT 1"),
 
+	/**
+	 * Kingbase.
+	 * @since 3.4.0
+	 */
+	KINGBASE("KingbaseES","com.kingbase8.Driver","com.kingbase8.xa.KBXADataSource","SELECT 1") {
```

Comment:
There is still an issue with the formatting. Please refer to the [Spring Boot Contributing Guidelines](https://github.com/spring-projects/spring-boot/blob/main/CONTRIBUTING.adoc).

Please use `./gradlew format` to reformat your code.